.github/workflows/kind_e2e.yaml

```diff
@@ -6,6 +6,12 @@
 name: E2E - KinD
 
 on:
+  push:
+    branches:
+      - master
+      - release-*
+  schedule:
+    - cron: '0 */3 * * *'
```

Comment:
@artursouza @pkedy Does the schedule seem ok for the e2e tests?

Reply:
We run every 2 hours in dapr/dapr but this one is OK for me.